src/The_Cartridge_Header.md

```diff
@@ -127,145 +133,150 @@ Code | Publisher
-
-## 0147 - Cartridge Type
-
-Specifies which Memory Bank Controller (if any) is used in the
-cartridge, and if further external hardware exists in the cartridge.
-
-Code | Type
------|------
- $00 | ROM ONLY
- $01 | MBC1
- $02 | MBC1+RAM
- $03 | MBC1+RAM+BATTERY
- $05 | MBC2
- $06 | MBC2+BATTERY
- $08 | ROM+RAM [^rom_ram]
- $09 | ROM+RAM+BATTERY [^rom_ram]
- $0B | MMM01
- $0C | MMM01+RAM
- $0D | MMM01+RAM+BATTERY
- $0F | MBC3+TIMER+BATTERY
- $10 | MBC3+TIMER+RAM+BATTERY [^mbc30]
- $11 | MBC3
- $12 | MBC3+RAM [^mbc30]
- $13 | MBC3+RAM+BATTERY [^mbc30]
- $19 | MBC5
- $1A | MBC5+RAM
- $1B | MBC5+RAM+BATTERY
- $1C | MBC5+RUMBLE
- $1D | MBC5+RUMBLE+RAM
- $1E | MBC5+RUMBLE+RAM+BATTERY
- $20 | MBC6
- $22 | MBC7+SENSOR+RUMBLE+RAM+BATTERY
- $FC | POCKET CAMERA
- $FD | BANDAI TAMA5
- $FE | HuC3
- $FF | HuC1+RAM+BATTERY
+## 0146 – SGB Flag
+
+This byte specifies whether the game supports SGB functions.
+The SGB will ignore any [packets](<#Command Packet Transfers>) if this byte is set to a value other than $03 (which is typically $00).
+
+## 0147 – Cartridge Type
+
+This byte indicates what kind of hardware is present on the cartridge, notably its [mapper](#MBCs).
+
+Code  | Type
+------|------
+`$00` | ROM ONLY
+`$01` | MBC1
+`$02` | MBC1+RAM
+`$03` | MBC1+RAM+BATTERY
+`$05` | MBC2
+`$06` | MBC2+BATTERY
+`$08` | ROM+RAM [^rom_ram]
+`$09` | ROM+RAM+BATTERY [^rom_ram]
+`$0B` | MMM01
+`$0C` | MMM01+RAM
+`$0D` | MMM01+RAM+BATTERY
+`$0F` | MBC3+TIMER+BATTERY
+`$10` | MBC3+TIMER+RAM+BATTERY [^mbc30]
+`$11` | MBC3
+`$12` | MBC3+RAM [^mbc30]
+`$13` | MBC3+RAM+BATTERY [^mbc30]
+`$19` | MBC5
+`$1A` | MBC5+RAM
+`$1B` | MBC5+RAM+BATTERY
+`$1C` | MBC5+RUMBLE
+`$1D` | MBC5+RUMBLE+RAM
+`$1E` | MBC5+RUMBLE+RAM+BATTERY
+`$20` | MBC6
+`$22` | MBC7+SENSOR+RUMBLE+RAM+BATTERY
+`$FC` | POCKET CAMERA
+`$FD` | BANDAI TAMA5
+`$FE` | HuC3
+`$FF` | HuC1+RAM+BATTERY
 
 [^rom_ram]:
-No licensed cartridge makes use of this option. Exact behaviour is unknown.
+No licensed cartridge makes use of this option. The exact behavior is unknown.
 
 [^mbc30]:
-MBC3 with RAM size 64 KByte refers to MBC30, used only in _Pocket Monsters Crystal Version_ for Japan.
-
-## 0148 - ROM Size
-
-Specifies the ROM Size of the cartridge. Typically calculated as "N such that 32 KiB
-<< N".
-
-Code | Size      | Amount of banks
------|-----------|-----------------
- $00 |  32 KByte | 2 (No ROM banking)
- $01 |  64 KByte | 4
- $02 | 128 KByte | 8
- $03 | 256 KByte | 16
- $04 | 512 KByte | 32
- $05 |   1 MByte | 64
- $06 |   2 MByte | 128
- $07 |   4 MByte | 256
- $08 |   8 MByte | 512
- $52 | 1.1 MByte | 72 [^weird_rom_sizes]
- $53 | 1.2 MByte | 80 [^weird_rom_sizes]
- $54 | 1.5 MByte | 96 [^weird_rom_sizes]
+MBC3 with 64 KiB of SRAM refers to MBC30, used only in _Pocket Monsters: Crystal Version_ (the Japanese version of _Pokémon Crystal Version_).
+
+## 0148 – ROM Size
+
+This byte specifies the size of the ROM on the cartridge. In most cases the ROM size
+is given by `32 KiB x (2 ^ <value>)`:
```

Comment:
```suggestion
This byte indicates how much ROM is present on the cartridge.
In most cases, the ROM size is given by `32 KiB × (1 << <size>)`:
```

Reply:
I'm not sure the shift expression is an improvement. 🤔

Reply:
The "power" operator like this is ambiguous with XOR. A double-star might work if you think shifting is not a good idea.

Reply:
Good point, I didn't think of that. Let's go with the shift then.